apps/aecore/src/aec_blocks.erl

```diff
@@ -420,7 +420,6 @@ validate_key_block(#key_block{} = Block) ->
 validate_micro_block(#mic_block{} = Block) ->
     Validators = [fun validate_txs_hash/1,
                   fun validate_gas_limit/1,
```

Comment:
Out of scope for this PR, looks in scope of ["Hook gas limit per microblock to gas for all transactions"](https://www.pivotaltracker.com/story/show/160392862):
* The check on microblock gas limit may need review - at the moment for e.g. call tx the gas limit is considered so a call tx with a large gas limit though little gas used seems exhausting space for txs in microblock while spending only the gas used.
* The conductor, in the DB transaction performing both block insertion and checks, [does not perform checks against block gas limit](https://github.com/aeternity/epoch/blob/e0bfd9e4901a0c1a98c89c172f54f9d726475e35/apps/aecore/src/aec_trees.erl#L417-L418). That check seems performed before calling transaction though (1) I would have expected check inside DB transaction and (2) I have not checked all code paths for that.